src/js/components/TaskMesosUrlComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var InfoStore = require("../stores/InfoStore");
+
+var TaskMesosUrlComponent = React.createClass({
+  displayName: "TaskMesosUrlComponent",
+  propTypes: {
+    className: React.PropTypes.string,
+    task: React.PropTypes.object.isRequired,
+    text: React.PropTypes.string
+  },
+
+ getInitialState: function () {
+    return {
+      info: InfoStore.info
+    };
+  },
+
+  componentDidMount: function () {
+    InfoActions.requestInfo();
+  },
+
+  componentWillMount: function () {
+    InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
+  },
+
+  onInfoChange: function () {
+    this.setState({
+      info: InfoStore.info
+    });
+  },
+
+  buildUrl: function (task, info) {
```

Comment:
I'm not sure why we need a whole component for this. Could we expose this function in the `InfoStore` and make use of that it `TaskListItemComponent`? If not I would suggest moving this function to `TaskListItemComponent`